libs/sdk-core/src/persist/reverseswap.rs

```diff
@@ -57,17 +66,42 @@ impl SqliteStorage {
 
     pub(crate) fn list_reverse_swaps(&self) -> PersistResult<Vec<FullReverseSwapInfo>> {
         let con = self.get_connection()?;
-        let mut stmt = con.prepare(&self.select_reverse_swap_query())?;
+        let mut stmt = con.prepare(&self.select_reverse_swap_query(""))?;
```

Comment:
I had a look into `persist/swap.rs` and there a string slice is used in a very similar method. The difference is that instead of an empty string, `"true"` is being passed, which makes the intent of the `where_clause` parameter more clear. I've applied the same approach in [100efb1](https://github.com/breez/breez-sdk/pull/728/commits/100efb106c287e41c5bbf5248ae84ea4ada85064) but I can also change to an optional string in both places if that makes sense.